config/reports/ukraineSubsidyOproep1Report.js

```diff
@@ -87,26 +86,23 @@ export default {
     const queryResponse = await batchedQuery(queryString);
     const data = queryResponse.results.bindings.map((subsidie) => {
       return {
-        subsidieURI: getSafeValue(subsidie, "subsidie"),
-        modified: getSafeValue(subsidie, "modified"),
-        status: getSafeValue(subsidie, "status"),
-        bestuurseenheid: getSafeValue(subsidie, "bestuurseenheid"),
-        kbo: getSafeValue(subsidie, "kbo"),
```

Comment:
Removing the double quotes from the KBO field causes the 0 on the left to be truncated (e.g., a kbo of `0xyz` becomes `xyz`) since I think it gets interpreted as a number.

Maybe we should keep the double quote wrapping on this one?